couchbase-sys/Makefile

```diff
@@ -0,0 +1,31 @@
+binding:
```

Comment:
For my understanding why do we need this makefile? `cargo build` should run all the bindgen bits automatically.